spec/dummy/app/assets/templates/hoge.ja.html.erb

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>hoge.ja.html.erb</title>
```

Comment:
Could you use an English metasyntactic variable instead of Japanese one?
ref. https://en.wikipedia.org/wiki/Metasyntactic_variable